test/core/TriggerOrderManager.js

```diff
@@ -35,43 +35,18 @@ describe("TriggerOrderManager", function () {
     let jpy
     let usdc
     let usdt
-    let btcPriceFeed
-    let ethPriceFeed
-    let dogePriceFeed
-    let gbpPriceFeed
-    let eurPriceFeed
-    let jpyPriceFeed
-    let usdcPriceFeed
-    let usdtPriceFeed
-    let vlpPriceFeed
-    let vaultPriceFeed
     let cooldownDuration
+    let pyth
```

Comment:
this variable seems no assignment?

Reply:
for testing, we cant pass pyth. pyth provide only for mainnet and test networks, not for hardhat test.
so I left it as empty for now.